### PR TITLE
Dịch "weight-decay_vn" - Phần 6

### LINKED ISSUE #930: Dịch "weight-decay_vn" - Phần 6
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/weight-decay_vn.md#L410-L449

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by azraelzhor**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #930

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 1]
```diff
@@ -434,7 +434,11 @@ In this book we will default to the simple heuristic of applying weight decay on
 
 <!-- ===================== Bắt đầu dịch Phần 6 ===================== -->
 
-*dịch đoạn phía trên*
+Đến giờ, chúng ta chỉ đề cập đến một khái niệm về những gì được xem là một hàm *tuyến tính* đơn giản.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Tới giờ, chúng ta mới chỉ đề cập đến một ý niệm về những gì cấu thành nên một hàm *tuyến tính* đơn giản.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 2]
```diff
@@ -434,7 +434,11 @@ In this book we will default to the simple heuristic of applying weight decay on
...
+Hơn nữa, những gì được xem là một hàm *phi tuyến* đơn giản, có thể là một câu hỏi thậm chí còn phức tạp hơn.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Hơn nữa, những gì cấu thành nên một hàm *phi tuyến* đơn giản lại là một câu hỏi thậm chí còn phức tạp hơn.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 3]
```diff
@@ -434,7 +434,11 @@ In this book we will default to the simple heuristic of applying weight decay on
...
+Ví dụ, [Tái tạo các không gian kernel Hilbert (RKHS)](https://en.wikipedia.org/wiki/Reproducing_kernel_Hilbert_space) cho phép chúng ta áp dụng các công cụ được giới thiệu cho các hàm tuyến tính trong một ngữ cảnh phi tuyến.
+Không may mắn thay, các giải thuật dựa vào RKHS có xu hướng mở rộng quy mô hoàn toàn khi dữ liệu lớn, đa chiều.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Không may mắn thay, các giải thuật dựa vào RKHS thường không thể nhân rộng và hoạt động hiệu quả trên dữ liệu lớn, đa chiều.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 4]
```diff
@@ -434,7 +434,11 @@ In this book we will default to the simple heuristic of applying weight decay on
...
+Trong quyển sách này chúng ta sẽ mặc định dùng phương pháp heuristic đơn giản áp dụng phân rã trọng số trên tất cả các tầng của mạng học sâu.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Dựa trên một thực nghiệm đơn giản, trong quyển sách này chúng ta mặc định sẽ áp dụng phương pháp suy giảm trọng số cho tất cả các tầng của mạng học sâu.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 5]
```diff
@@ -449,9 +453,11 @@ In this book we will default to the simple heuristic of applying weight decay on
 * You can have different optimizers within the same training loop, e.g., for different sets of parameters.
 -->
 
-*dịch đoạn phía trên*
-
-
+* Điều chuẩn là một phương pháp phổ biến để giải quyết quá khớp.
+Nó thêm một lượng phạt vào hàm mất mát trong tập huấn luyện để giảm thiểu độ phức tạp của mô hình học.
+* Một lựa chọn cụ thể để giữ mô hình đơn giản là phân rã trọng số sử dụng lượng phạt $\ell_2$. Điều này dẫn đến phân rã trọng số trong các bước cập nhật của giải thuật học.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
* Một cách cụ thể để giữ mô hình đơn giản là suy giảm trọng số sử dụng lượng phạt $\ell_2$. Điều này dẫn đến giá trị trọng số sẽ suy giảm trong các bước cập nhật của giải thuật học.
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 6]
```diff
@@ -469,7 +475,18 @@ In addition to weight decay, increased training, and the use of a model of suita
 6. In Bayesian statistics we use the product of prior and likelihood to arrive at a posterior via $P(w \mid x) \propto P(x \mid w) P(w)$. How can you identify $P(w)$ with regularization?
 -->
 
-*dịch đoạn phía trên*
+1. Thí nghiệm với giá trị của $\lambda$ trong bài toán ước lượng ở trang này.
+Vẽ đồ thị độ chính xác của tập huấn luyện và tập kiểm tra như một hàm số của $\lambda$.
+Bạn quan sát được những gì?
+2. Sử dụng tập kiểm định để tìm giá trị tối ưu của $\lambda$.
+Nó có thật sự là giá trị tối ưu hay không?
+Điều này có vấn đề gì không?
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Điều này có quan trọng không?
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 7]
```diff
@@ -469,7 +475,18 @@ In addition to weight decay, increased training, and the use of a model of suita
...
+3. Các phương trình cập nhật sẽ có dạng như thế nào nếu thay vì $\|\mathbf{w}\|^2$ chúng ta sử dụng lượng phạt $\sum_i |w_i|$ (được gọi là điều chuẩn $\ell_1$).
+4. Chúng ta đã biết rằng $\|\mathbf{w}\|^2 = \mathbf{w}^\top \mathbf{w}$.
+Bạn có thể tìm một phương trình tương tự cho các ma trận (các nhà toán học gọi nó là [chuẩn Frobenius](https://en.wikipedia.org/wiki/Matrix_norm#Frobenius_norm)) hay không?
+5. Ôn lại mối quan hệ giữa lỗi huấn luyện và lỗi khái quát.
+Bên cạnh phân rã trọng số, huấn luyện tăng cường, và việc sử dụng một mô hình có độ phức tạp phù hợp, bạn có thể nghĩ ra cách nào khác để giải quyết quá khớp không?
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Bên cạnh phân rã trọng số, tăng cường việc huấn luyện và sử dụng một mô hình có độ phức tạp phù hợp, bạn có thể nghĩ ra cách nào khác để giải quyết quá khớp không?
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 3 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay @lkhphuc

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 4 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 5 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 7 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 6 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: cái này dịch kiểu nào cũng được đúng không @lkhphuc ?

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 2 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 1 above)
Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: khái niệm vs ý niệm có khác nhau nhiều không nhỉ ?

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 6 above)
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Mình offer thay đổi của @lkhphuc , câu dịch gốc mình thấy khá giống văn nói và đọc cũng cấn.

Comment (azraelzhor) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: okay @duythanhvn

File: chapter_multilayer-perceptrons/weight-decay_vn.md (hunk 1 above)
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/weight-decay_vn.md]: Có đấy, khái niệm là `concept`, ý niệm là `notion`.

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 8]
```diff
@@ -469,7 +475,18 @@ In addition to weight decay, increased training, and the use of a model of suita
...
+Vẽ đồ thị độ chính xác của tập huấn luyện và tập kiểm tra như một hàm số của $\lambda$.
+Bạn quan sát được những gì?
+2. Sử dụng tập kiểm định để tìm giá trị tối ưu của $\lambda$.
+Nó có thật sự là giá trị tối ưu hay không?
+Điều này có quan trọng không?
+3. Các phương trình cập nhật sẽ có dạng như thế nào nếu thay vì $\|\mathbf{w}\|^2$ chúng ta sử dụng lượng phạt $\sum_i |w_i|$ (được gọi là điều chuẩn $\ell_1$).
+4. Chúng ta đã biết rằng $\|\mathbf{w}\|^2 = \mathbf{w}^\top \mathbf{w}$.
+Bạn có thể tìm một phương trình tương tự cho các ma trận (các nhà toán học gọi nó là [chuẩn Frobenius](https://en.wikipedia.org/wiki/Matrix_norm#Frobenius_norm)) hay không?
+5. Ôn lại mối quan hệ giữa lỗi huấn luyện và lỗi khái quát.
+Bên cạnh phân rã trọng số, tăng cường việc huấn luyện và sử dụng một mô hình có độ phức tạp phù hợp, bạn có thể nghĩ ra cách nào khác để giải quyết quá khớp không?
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
Bên cạnh suy giảm trọng số, tăng cường việc huấn luyện và sử dụng một mô hình có độ phức tạp phù hợp, bạn có thể nghĩ ra cách nào khác để giải quyết quá khớp không?
```

File: chapter_multilayer-perceptrons/weight-decay_vn.md [hunk 9]
```diff
@@ -516,4 +533,4 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 6 -->
-*
+* Lê Cao Thăng
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/weight-decay_vn.md]:
```suggestion
* Lê Cao Thăng
* Lê Khắc Hồng Phúc
```
